MEV_spec_RPC_v0_2.md

```diff
@@ -0,0 +1,141 @@
+---
+tags: spec
+---
+
+# MEV-Geth RPC v0.1
+
+# eth_sendBundle
+
+### Description
+
+Sends a bundle of transactions to the miner. The bundle has to be executed at the beginning of the block (before any other transactions), with bundle transactions executed exactly in the same order as provided in the bundle. During the Flashbots Alpha this is only called by the MEV-relay.
```

Comment:
MEV-relay? Are we calling it that? Flashbots relay?

Reply:
Fixed